src/main/java/com/epam/task/first/logic/ArrayLogic.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.task.first.logic;
+
+import com.epam.task.first.entities.Array;
+
+import java.util.ArrayList;
+
+public class ArrayLogic {
+
+    final String EMPTY = "Массив пуст.";
+    final String WRONGING = "Данного индекса не существует.";
+    final String CORRECT = "Одобрено!";
+    final int ZERO = 0;
+
+
+    public final Integer findMax(final Array array) {
+        int maxValue = Integer.MIN_VALUE;
+        ArrayList<Integer> arrayList = array.getArray();
+        if (!arrayList.isEmpty()) {
+            for (Integer element : arrayList) {
+                if (element > maxValue) {
+                    maxValue = element;
+                }
+            }
+            return maxValue;
+        } else {
+            System.out.println(EMPTY);
+            return ZERO;
+        }
+    }
+
+    public final Integer findMin(final Array array) {
+        int minValue = Integer.MAX_VALUE;
+        ArrayList<Integer> arrayList = array.getArray();
+        if (!arrayList.isEmpty()) {
+            for (Integer element : arrayList) {
+                if (element < minValue) {
+                    minValue = element;
+                }
+            }
+            return minValue;
+        } else {
+            System.out.println(EMPTY);
+            return ZERO;
+        }
+
+    }
+
+    public final boolean setElement(final Array array, final Integer index, final Integer element) {
```

Comment:
you don't need this method, it's a logic class, you can't to print here, for this you have printer class